packages/govern-core/buidler.config.ts

```diff
@@ -0,0 +1,156 @@
+import { BuidlerConfig, task, usePlugin } from '@nomiclabs/buidler/config'
+import dotenv from 'dotenv'
+import { ethers } from 'ethers'
+import { readFileSync, writeFileSync } from 'fs'
+import {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals
+} from 'unique-names-generator'
+
+usePlugin('buidler-local-networks-config-plugin')
+usePlugin('buidler-typechain')
+usePlugin('solidity-coverage')
+usePlugin('@nomiclabs/buidler-ethers')
+usePlugin('@nomiclabs/buidler-etherscan')
+usePlugin('@nomiclabs/buidler-waffle')
+
+dotenv.config({ path: '../../.env' })
```

Comment:
Just noticed: this is getting the `.env` file from the root folder, while the above `.env.example` is on this folder. Should we move the `.env.example` to the root folder?